### ISSUE TITLE
Visual Recognition for Websites

### ISSUE THREAD

**Issue by JohnyFederal1337** [stackblitz/bolt.new#4697]
**Is your feature request related to a problem? Please describe:**

When i ask the ai to make me a website similar to another certain website or with the idea of it, the ai says it does make something like that website but the results never ending up looking like it, thats why it would be amazing to have a feature that could actually know what that website looks like and get the idea of it

**Describe the solution you'd like:**

Basically the ai being able to know what the website looks like and turning it into the prompt 

**Describe alternatives you've considered:**

Tried asking chatgpt for clear instructions on how the website is build and every spec about it (didnt go far)
